CMakeLists.txt

```diff
@@ -188,16 +188,11 @@ else()
     set(BMQ_TARGET_BMQTOOL_NEEDED YES)
   endif()
 
-  if (installBMQPLUGINS)
-    set(BMQ_TARGET_BMQBRKR_NEEDED    YES)
-    set(BMQ_TARGET_BMQBRKRCFG_NEEDED YES)
-    set(BMQ_TARGET_BMQTOOL_NEEDED    YES)
+  if (installPROMETHEUS)
     set(BMQ_TARGET_MWC_NEEDED        YES)
     set(BMQ_TARGET_BMQ_NEEDED        YES)
     set(BMQ_TARGET_MQB_NEEDED        YES)
```

Comment:
Wouldn't you also need `set(BMQ_TARGET_BMQBRKR_NEEDED   YES)` here?
I thought you also need to build the broker for the Prometheus plugin to work.